---
bool success
string message
geometry_msgs/TransformStamped Tec  # effector <- camera transformation
geometry_msgs/TransformStamped Twm  # world    <- marker transformation
